### PR TITLE
Add example transaction transferring child NFTs from parent account

### PR DESCRIPTION
**Pull request by sisyphusSmiling**
Closes: #176

### CODE REVIEW

File: transactions/hybrid-custody/send_child_nfts_with_parent.cdc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        self.provider = providerCap.borrow()!
+    }
+
+    execute {
+        // get the recipient's public account object
+        let recipient = getAccount(to)
+
+        // get a reference to the recipient's NFT Receiver
+        let receiverRef = recipient.capabilities.borrow<&{NonFungibleToken.Receiver}>(self.collectionData.publicPath)
+			?? panic("Could not borrow receiver reference to the recipient's Vault")
```
Comment (austinkline) [inline transactions/hybrid-custody/send_child_nfts_with_parent.cdc:43]: nit: indentation

File: transactions/hybrid-custody/send_child_nfts_with_parent.cdc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    // signer is the parent account
+    prepare(signer: auth(Storage) &Account) {
+        // get the manager resource and borrow childAccount
+        let m = signer.storage.borrow<auth(HybridCustody.Manage) &HybridCustody.Manager>(from: HybridCustody.ManagerStoragePath)
+            ?? panic("manager does not exist")
+        let childAcct = m.borrowAccount(addr: child) ?? panic("child account not found")
+        
+        self.collectionData = ExampleNFT.resolveContractView(resourceType: nil, viewType: Type<MetadataViews.NFTCollectionData>()) as! MetadataViews.NFTCollectionData?
+            ?? panic("Could not get the vault data view for ExampleNFT")
```
Comment (austinkline) [inline transactions/hybrid-custody/send_child_nfts_with_parent.cdc:22]: You can be more generic here and borrow against the `NonFungibleToken` contract if you want

**Review by austinkline — APPROVED** [review]

File: transactions/hybrid-custody/send_child_nfts_with_parent.cdc (hunk 2 above)
Comment (sisyphusSmiling) [reply transactions/hybrid-custody/send_child_nfts_with_parent.cdc:22]: True, I'll update this and the ft transaction to be generic before merging.

**Comment by sisyphusSmiling** [comment]
Updated transactions to be more generic. In the process, I realized the ExampleNFT implementations didn't conform to the NFT standard so updated those for testing purposes.

**Review by nvdtf — APPROVED** [review]

**Pull request merged**